test/pom.xml

```diff
@@ -27,14 +27,375 @@ THE SOFTWARE.
 
   <parent>
     <groupId>org.jenkins-ci.main</groupId>
-    <artifactId>jenkins-test-parent</artifactId>
+    <artifactId>jenkins-parent</artifactId>
     <version>${revision}${changelist}</version>
-    <relativePath>../test-pom</relativePath>
   </parent>
 
   <artifactId>jenkins-test</artifactId>
 
   <name>Tests for Jenkins core</name>
   <description>Functional tests for Jenkins core</description>
 
+  <properties>
+    <concurrency>2</concurrency> <!-- may use e.g. 2C for 2 × (number of cores) -->
+    <mavenDebug>false</mavenDebug>
+    <jacocoSurefireArgs /><!-- empty by default -->
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.jenkins-ci.main</groupId>
+        <artifactId>jenkins-bom</artifactId>
+        <version>${project.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+      <dependency>
+        <groupId>com.google.code.findbugs</groupId>
+        <artifactId>annotations</artifactId>
+        <version>3.0.1</version>
+        <scope>provided</scope>
+      </dependency>
```

Comment:
Wouldn't it be better to use the spotbugs annotations rather than findbugs?
```
<!-- https://mvnrepository.com/artifact/com.github.spotbugs/spotbugs-annotations -->
<dependency>
    <groupId>com.github.spotbugs</groupId>
    <artifactId>spotbugs-annotations</artifactId>
    <version>3.1.12</version>
    <scope>provided</scope>
</dependency>
```

Reply:
See PR description: I am not touching any details here, merely rearranging files. If you want to propose a change like that, it would better be done in a separate PR.